master/getting-started/kubernetes/installation/hosted/kubernetes-datastore/index.md

```diff
@@ -51,91 +50,135 @@ Ensure you have a cluster which meets the above requirements.  There may be addi
 > **Note**: There is currently no upgrade path to switch between
 > different installation options. Therefore, if you are upgrading
 > from Calico v2.1, use the
-> [Calico policy-only with user-supplied networking](#2-calico-policy-only-with-user-supplied-networking)
+> [Calico policy-only with user-supplied networking](#policy-only)
 > installation instructions to upgrade Calico policy-only which
 > leaves the networking solution unchanged.
 {: .alert .alert-info}
 
+### Before you start: if your cluster has RBAC enabled
 
-### RBAC
-
-Before you install {{site.prodname}}, if your Kubernetes cluster has RBAC enabled, you'll need to create the following
-RBAC roles to allow API access to {{site.prodname}}.
-
-Apply the following manifest to create these necessary RBAC roles and bindings.
-
-> **Note**: The following RBAC policy is compatible with the Kubernetes v1.7+
-> manifests only.
-{: .alert .alert-info}
+Install {{site.prodname}}'s RBAC manifest, which creates roles and role bindings for {{site.prodname}}'s components:
 
 ```
 kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/rbac-kdd.yaml
 ```
 
->[Click here to view the above yaml directly.](../rbac-kdd.yaml)
-
-### 1. {{site.prodname}} policy with {{site.prodname}} networking (Beta)
+>[Click here to view the above YAML directly.](../rbac-kdd.yaml)
 
-With Kubernetes as the {{site.prodname}} datastore, {{site.prodname}} has beta support for {{site.prodname}} networking.  This provides BGP-based
-networking with a full node-to-node mesh and/or explicit configuration of peers.
+### Option 1: {{site.prodname}} policy with {{site.prodname}} networking
 
-To install {{site.prodname}} with {{site.prodname}} networking, run one of the commands below based on your Kubernetes version.
-This will install {{site.prodname}} and will initially create a full node-to-node mesh.
+With Kubernetes as the {{site.prodname}} datastore, {{site.prodname}} supports 
+{{site.prodname}} networking.  This provides BGP-based networking with a full 
+node-to-node mesh and/or explicit configuration of peers.
 
-> **Note**: Calico v2.5.0 or later with Kubernetes backend requires Kubernetes
-> v1.7.0 or higher.
-{: .alert .alert-info}
-
-```
-kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/kubernetes-datastore/calico-networking/1.7/calico.yaml
-```
+To install {{site.prodname}} with BGP networking:
 
->[Click here to view the above yaml directly.](calico-networking/1.7/calico.yaml)
+1. Download [the {{site.prodname}} networking manifest](calico-networking/1.7/calico.yaml)
```

Comment:
I wonder if, rather than making Typha part of the "mainline" install process, it would be better to keep the one-line install command for ease-of-use, and then add a note box which says something like "If you're building a cluster larger than 50 nodes, we recommend you follow the steps to enable Typha".

I think that is going to be a smoother experience for more users, given most clusters are only a few nodes.

Reply:
Ugh, I jumped the other way, we were getting to the point where we had notes on top of notes :-(

Reply:
We also get a lot of folks missing the IP pool setting so I wanted to make that more prominent

Reply:
Yeah understood, I'm just worried this makes the process too burdensome for a first-time user who really just wants to copy and paste a command to get started. Maybe there's an in-between?

@emanic WDYT?

Reply:
Actually, taking a closer look, it would be installation option #4, looks like. And then you can add to the titles of the others (less than or equal to 50 nodes) or something like that.

Reply:
I'd like to not have two separate manifests here. We've already got too many.

This is just the difference between:
- Small clusters, which don't need any tweaking
- Large clusters, which will need tweaking no matter what, even with a separate manifest. 

We can do both in one manifest, and then just streamline the instructions so that for the "small" case (which covers 90% of users) no extra work is required. 

For those running larger clusters, the extra steps are clearly documented.